src/api/accounts/businesses.md

```diff
@@ -0,0 +1,140 @@
+---
+layout:         default
+grand_parent:   API Reference
+parent:         Accounts
+title:          Businesses
+permalink:      /api/businesses
+---
+
+# Businesses
+{:.no_toc}
+
+A Business represents a company registered with the New Zealand Companies Office.
+
+A Business is associated with a single [Account][].
+
+## Contents
+{:.no_toc .text-delta}
+
+* TOC
+{:toc}
+
+## Models
+
+### Business
+
+|     Field     |        Type        |                    Description                     |
+| :------------ | :----------------- | :------------------------------------------------- |
+| id            | String             | The unique identifier.                             |
+| accountId     | String             | The Centrapay accountId for an org account.        |
+| nzbn          | String             | The unique NZBN identifier.                        |
+| name          | String             | Trading name recorded in the Companies Register.   |
+| companyNumber | String             | Company number recorded in the Companies Register. |
+| createdAt     | {% dt Timestamp %} | When the Business was created.                     |
+| updatedAt     | {% dt Timestamp %} | When the Business was updated.                     |
+| createdBy     | {% dt CRN %}       | The User or API Key that created the Business.     |
+| updatedBy     | {% dt CRN %}       | The User or API Key that updated the Business.     |
+
+## Operations
+
+### Create a Business
+
+If `accountId` is not provided when creating a business, then a new
+org account will be created and associated to the business.
+
+{% reqspec %}
+  POST '/api/businesses'
+  auth 'api-key'
+  body ({
+    nzbn: '9429046246448'
+  })
+{% endreqspec %}
+
+{% h4 Required Fields %}
+
+| Field |  Type  |         Description         |
+| :---- | :----- | :-------------------------- |
+| nzbn  | String | The unique NZBN identifier. |
+
+{% h4 Optional Fields %}
+
+|   Field   |  Type  |       Description        |
+| :-------- | :----- | :----------------------- |
+| accountId | String | The Centrapay accountId. |
+
+{% h4 Example response payload %}
+
+```json
+{
+  "id": "DKTs3U38hdhfEqwF1JKoT2",
+  "accountId": "Jaim1Cu1Q55uooxSens6yk",
+  "nzbn": "9429046246448",
+  "name": "CENTRAPAY LIMITED",
+  "companyNumber": "6340244",
+  "createdAt": "2020-06-12T01:17:46.499Z",
+  "updatedAt": "2020-06-12T01:17:46.499Z",
+  "createdBy": "crn:WIj211vFs9cNACwBb04vQw:api-key:MyApiKey",
+  "updatedBy": "crn:WIj211vFs9cNACwBb04vQw:api-key:MyApiKey",
+}
+```
+
+{% h4 Error Responses %}
+
+| Status |         Code          |                            Description                            |
+| :----- | :-------------------- | :---------------------------------------------------------------- |
+| 403    | ACCOUNT_NOT_SUPPORTED | The Centrapay account provided is not supported for this request. |
```

Comment:
`INVALID_ACCOUNT` : Account does not exist, is not authorized or is the wrong type.

Reply:
Changed it :)